.github/actions/e2e/install-eksctl/action.yaml

```diff
@@ -0,0 +1,16 @@
+name: InstallEKSCTL
```

Comment:
I'm worried about how many different ways this opens us up to failures. 

Looking briefly for some ways to use ECR images, I saw this https://agileek.github.io/software/aws/2020/10/28/using-an-ecr-image-in-github-actions/#:~:text=The%20complete%20GitHub%20workflow%3A but I don't know how viable it is.

Reply:
I feel the effort is worth it to just use the tools image we already have, rather than creating some other tool installation containers. WDYT?

Reply:
> I'm worried about how many different ways this opens us up to failures

I'm curious what you're thinking that these failures could be. This is the prescribed download method for `eksctl`

The tools image that we have doesn't currently work out-of-the-box and needs some tweaking to get it working properly with GHA. I can take that as an AI and fast-follow but I'm not convinced that this needs to be blocking for this PR to go in.

Reply:
> I'm curious what you're thinking that these failures could be.

Really just thinking about unexpected bash issues. I'm fine to merge this and do a fast follow, but don't want us to forget about this.

Reply:
Sounds good, not planning to forget about this. Part of the follow-up for this is going to be creating the testing image for GHA to run the tests with